pyproject.toml

```diff
@@ -280,6 +272,7 @@ exclude=[
 "__init__.py" = ["E402", "F401", "F403"]
 "astropy/io/registry/compat.py" = ["F822"]
 "astropy/modeling/models.py" = ["F401", "F403", "F405"]
+"astropy/utils/decorators.py" = ["D214", "D215"]  # keep Examples section indented.
```

Comment:
How about making those `Examples` proper docstring ones (i.e., add `>>>` before, etc.? Or is the ones with `>>>` already?

Reply:
Yeah, the problem is lines like this

```
        >>> help(add) # doctest: +SKIP
        Help on function add in module __main__:
        <BLANKLINE>
        add(num1, num2)
            Perform addition.
            Parameters
            ----------
            num1, num2 : Numbers
            Returns
            -------
            result : Number
```

The Parameters & Returns both want to be dedented.

Reply:
OK, maybe easiest to just skip it for now indeed.